### PR TITLE
Adds pagination on admin listing of posts

### PR DESCRIPTION
**Pull request by caitlingoldman**
Adds pagination to the table of posts in the admin section. Defaults to 50 per page or you can set using a new config option: `admin_posts_per_page`.

### CODE REVIEW

File: app/models/monologue/post.rb [hunk 1]
```diff
@@ -44,6 +44,14 @@ def self.page p
     self.limit(per_page).offset(offset)
   end
 
+  def self.listing_page(p)
+    per_page = Monologue::Config.admin_posts_per_page || 50
+    set_total_pages(per_page)
+    p = (p.nil? ? 0 : p.to_i - 1)
+    offset =  p * per_page
```
Comment (msevestre) [inline app/models/monologue/post.rb]: could you create a private method and reuse it. That code is exactly the same as the one in the method self.page.
Thanks

File: config/routes.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
   namespace :admin, path: "monologue" do
     get "/" => "posts#index", as:  "" # responds to admin_url and admin_path
+    get "/page/:page", to:  "posts#index", as:  "posts_page"
```
Comment (msevestre) [inline config/routes.rb]: Does it create admin_posts_page?

Comment (caitlingoldman) [inline config/routes.rb]: `admin_posts_page_path` I wanted to keep the same format ( `posts_page_path` ) that already exists in the non-admin section. Here's the new route and helper:

screen shot 2015-02-13 at 9 21 46 am

File: lib/monologue/configuration.rb [hunk 3]
```diff
@@ -3,34 +3,35 @@ class Configuration
...
-                    :twitter_locale,
+                  :twitter_username,
+                  :twitter_locale,
 
-                    :facebook_like_locale,
-                    :facebook_url,
-                    :facebook_logo, #used in the open graph protocol to display an image when a post is liked
+                  :facebook_like_locale,
+                  :facebook_url,
+                  :facebook_logo, #used in the open graph protocol to display an image when a post is liked
```
Comment (caitlingoldman) [inline lib/monologue/configuration.rb]: Someone else's comment. Didn't want to delete.

**Comment by caitlingoldman** [comment]
@msevestre not sure what changed, but my build is failing after merging master. Looking at the travic-ci It seems like the `responders` gem needs to be added to the gemfile? I had to do this locally to get it to work also. 

screen shot 2015-02-13 at 11 03 49 am

**Pull request merged**